### ISSUE TITLE
Calculation issue?

### ISSUE THREAD

**Issue by obra** [skalnik/aqi-wtf#80]
This morning in the East Bay, our numbers appeared to be wildly inflated. They looked to correlate with the uncorrected PurpleAir numbers. ￼ I was seeing numbers in the 130s when I believe I should have been seeing numbers in the 90s. At this moment, they look good￼

**Comment by lenscritic** [comment]
Agreed. If I go to aqi.wtf it shows 455 (!), but if I click the "a sensor 0km away" link it shows 36
